src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SymbolModel/ComputedSymbol.cs

```diff
@@ -30,6 +30,12 @@ string ISymbolModel.Replaces
             set { }
         }
 
+        string ISymbolModel.FileRename
```

Comment:
this looks funky, I'll have to look deeper to get more context

Reply:
This prevents to use property as ComputedSymbol class, but allows to use it as ISymbolModel.